server/models/ProfileModel.js

```diff
@@ -0,0 +1,69 @@
+const mongoose = require("mongoose");
+const bcrypt = require("bcryptjs");
+
+const profileSchema = new mongoose.Schema({
+  userId: {
+    type: String,
+    required: true,
+  },
+  username: {
+    type: String,
+    required: true,
+    unique: true,
+  },
+  firstName: {
+    type: String,
+    required: true,
+  },
+  lastName: {
+    type: String,
+    required: true,
+  },
+  email: {
+    type: String,
+    required: true,
+    unique: true,
+  },
+  password: {
+    type: String,
+    required: true,
+  },
+  register_date: {
+    type: Date,
+    default: Date.now,
+  },
+  admin: {
+    type: Boolean,
+    default: false,
+  },
+  address: String,
+  city: String,
+  phone: String,
+  profileImage: String,
+  photos: [],
+  jop: String,
+  description: String,
+  birth: String,
+  gender: String,
+  availability: Boolean,
+  rating: Number,
```

Comment:
Eventually we will be creating a separate model for Reviews/Rating, please exclude this field here

Reply:
Ok 
Thanks